common/frame_session.go

```diff
@@ -352,8 +352,11 @@ func (fs *FrameSession) parseAndEmitWebVitalMetric(object string) error {
 	})
 
 	_, span := TraceEvent(
-		fs.ctx, fs.targetID.String(), "web_vital", wv.SpanID,
-		trace.WithAttributes(attribute.Float64(wv.Name, value), attribute.String("rating", wv.Rating)))
+		fs.ctx, fs.targetID.String(), "web_vital", wv.SpanID, trace.WithAttributes(
+			attribute.String("web_vital.name", wv.Name),
```

Comment:
Notice that there is a considerable change here, as previously we were using the Web Vital name itself as attribute key.